blog/2024/04-28-jellyfin-release-10.9.0/index.mdx

```diff
@@ -0,0 +1,134 @@
+
+### General
+
+* We now support "trickplay" a.k.a. live video scrubbing. When scrubbing through a video with this enabled, you will be able to see a live preview of the video at that timestamp. Note that this requires explicit client support, which may require some time to become available depending on your client.
+
+* Web redirections are now handled more reliably, reducing the likelihood of hitting dead or invalid pages in the UI. Please ensure you force-refresh or clear any browser cache for your Jellyfin instance to see the full benefits.
+
+* DLNA support is now provided by a plugin and has been removed from the core server. This has several major benefits: first, it can be updated independently of the server to some degree; second, it will not be enabled for anyone unless they want it and explicitly install the plugin; and third, it reduces the potential of security holes around DLNA due to both previous benefits. If you wish to use (or continue to use) DLNA, please install the plugin after upgrading.
+
+* We now support AVIF and WEBP images for Pictures libraries.
+
+* Tags are now accounted for during searches, allowing one to search by tag.
+
+* We now support multiple simultaneous subtitle tracks (maximum of two, a primary and secondary) in the web player.
+
+* We've revamped the administrative dashboard UI to help improve usability and ease of finding options.
+
+### API & Security
+
+* All API endpoints now return proper return codes, ensuring that API endpoint results can be reliably interpreted without additional parsing.
+
+* Our API versioning scheme is now based off of the "last unstable build before a release" and uses a date-based version going forward, to help make it a bit clearer and assist in automating the circular dependencies in our API specifications.
+
+* Parental ratings are significantly improved, with better enforcement, inheritable ratings, and more.
+
+* LiveTV and Collection permissions are now discrete and configurable per-user.
+
+* The EasyPassword (PIN) feature has been removed as this was a big security risk especially for administrator accounts; QuickConnect login is still supported however.
+
+* User permission handling has been unified and numerous bugs fixed, ensuring a more secure server from untrusted users.
+
+### Core Server & Networking
+
+* The underlying .NET framework has been upgraded to version 8, the latest major and LTS release. This brings us many under-the-hood improvements and changes to help with our ongoing effort to streamline the codebase and improve overall performance.
+
+* Our minimum FFmpeg version has been bumped from 4.0 to 4.4 to take advantage of many new features. For users of our Jellyfin FFmpeg build (Docker, `.deb`/Debuntu repo, and Windows installs), you're already on an even newer version now with FFmpeg 6.0.1, but this minimum change would affect any other platforms.
```

Comment:
```suggestion
* Our minimum FFmpeg version has been bumped from 4.0 to 4.4 to take advantage of many new features. For users of our Jellyfin FFmpeg build (Docker, `.deb`/Debuntu repo, and Windows installs), you're already on a newer version with FFmpeg 6.0.1, but this minimum change would affect any other platforms.
```
I personally find this easier to read. I had to read the original multiple times 😅

Reply:
Yea that wording is attrocious, classic 3AM writing from me :D